src/module/headers.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-module-docstring
+from typing import Any
+from typing import Dict
+
+
+class Header:
+    """Header class to perform other header operations."""
+
+    def __init__(self, headers: Dict[str, Any]):
+        """
+        Initialize the header with default headers to be used.
+
+        :param headers: default headers to be used
+        """
+        self.headers = headers
+
+    def prepare_headers(self, headers: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        To concat the requesting headers with the default headers.
+
+        It will keep the exiting one the update the new one with new value
+
+        :param headers: headers pass as argument of the all requests
+        :return:
+        """
+        result_headers = self.headers
+        default_keys = self.headers.keys()
+        for key in headers.keys():
+            if key in default_keys:
+                result_headers[key] = headers[key]
+        return result_headers
```

Comment:
```suggestion
        self.headers.update(headers)
```

`self.headers` is a dictionary, dictionaries update function is capable to check whether key already present or not, and if it exists, it will update it's value and if it does not exists, it will add key with it's respective value.